projects/packages/jetpack-mu-wpcom/src/features/wpcom-site-menu/wpcom-site-menu.php

```diff
@@ -87,14 +87,16 @@ function wpcom_add_wpcom_menu_item() {
 		null
 	);
 
-	add_submenu_page(
-		$parent_slug,
-		esc_attr__( 'Add-ons', 'jetpack-mu-wpcom' ),
-		esc_attr__( 'Add-ons', 'jetpack-mu-wpcom' ),
-		'manage_options',
-		esc_url( "https://wordpress.com/add-ons/$domain" ),
-		null
-	);
+	if ( defined( 'IS_WPCOM' ) && IS_WPCOM ) {
```

Comment:
I see, this is the checking that we can do in https://github.com/Automattic/jetpack/pull/36060

Reply:
TIL